rsyncd/rsyncd.go

```diff
@@ -396,3 +408,14 @@ func (s *Server) Serve(ctx context.Context, ln net.Listener) error {
 		}()
 	}
 }
+
+func validateModule(mod Module) error {
+	if mod.Name == "" {
+		return errors.New("module has no name")
+	}
+	if mod.Path == "" {
+		return fmt.Errorf("module '%s' has empty path", mod.Name)
+	}
+
```

Comment:
What do you think, should we validate ACL as non-nil here as well?

The idea: you've to explicitly opt-in to `ACL: []string{}` to have "no security" (so nil is considered an error)

Does this idea make any sense? It sounds a bit unconventional though..

Reply:
For the moment, I don’t think we need that level of validation, but I wouldn’t rule it out in the future either.

In general, I’m not a fan of `[]string{}` behaving differently from `nil`. If we later wanted to add this validation, I’d rather suggest looking at `len(acls) == 0` and having a separate field that confirms the user knows what they’re doing.

Reply:
Yeah it sounds dirty, I agree. Let's leave that for future